games/devtest/mods/lighting/set_lighting.lua

```diff
@@ -0,0 +1,142 @@
+
+local lighting_sections = {
+	{n = "shadows", d = "Shadows",
+		entries = {
+			{ n = "intensity", d = "Shadow Intensity", min = 0, max = 1 }
+		}
+	},
+	{
+		n = "exposure", d = "Exposure",
+		entries = {
+			{n = "luminance_min", d = "Minimum Luminance", min = -10, max = 10},
+			{n = "luminance_max", d = "Maximum Luminance", min = -10, max = 10},
+			{n = "exposure_correction", d = "Exposure Correction", min = -10, max = 10},
+			{n = "speed_dark_bright", d = "Bright light adaptation speed", min = -10, max = 10, type="log2"},
+			{n = "speed_bright_dark", d = "Dark scene adaptation speed", min = -10, max = 10, type="log2"},
+			{n = "center_weight_power", d = "Power factor for center-weighting", min = 0.1, max = 10},
+		}
+	},
+}
+
+local function dump_lighting(lighting)
```

Comment:
Again, why is `dump` / `dump2` not an option?

Reply:
The original author of `/set_lighting` should know.
https://github.com/minetest/minetest/blob/180ec92ef982d9fb5c6bdc789f381335f77823c1/games/devtest/mods/lighting/init.lua#L20